APMrover2/AP_MotorsUGV.cpp

```diff
@@ -401,33 +401,25 @@ bool AP_MotorsUGV::pre_arm_check(bool report) const
         SRV_Channels::function_assigned(SRV_Channel::k_throttleRight) &&
         SRV_Channels::function_assigned(SRV_Channel::k_throttle) &&
         SRV_Channels::function_assigned(SRV_Channel::k_steering)) {
-        if (report) {
-            gcs().send_text(MAV_SEVERITY_CRITICAL, "PreArm: regular AND skid steering configured");
-        }
+        rover.arming.check_failed(ARMING_CHECK_NONE, report, "regular AND skid steering configured");
```

Comment:
we should probably be passing back a failure string.. it's OK for now but..

Reply:
Should we have entry points into the motors library for each different bit which might be checked 'though?

`So motors.prearm_checks_parameters(...)` `motors.prearm_checks_not_really_covered_by_other_bits` etc?

MdB has already said the same thing as you, so I might just drop the patches for now.

Reply:
sure, leaving it out seems fine to me.

Reply:
Dropped those.  We should come back to them.